Deferred/Deferred.swift

```diff
@@ -135,7 +205,15 @@ extension Deferred {
         }
         return d
     }
-
+    
+    /**
+    Transforms the deferred value once it becomes determined.
+    
+    :param: queue A dispatch queue for executing the transform on.
+    :param: transform A function to create something using the deferred value.
+    :returns: A new deferred value that is determined once the recieving
```

Comment:
"recieving" - I'd recommend just running a gsub across the whole project for "reciev" => "receiv" to catch all these